core/modules/filters/x-listops.js

```diff
@@ -186,4 +204,33 @@ Extended filter operators to manipulate the current list.
         }, []);
         return set;
     };
+
+	/*
```

Comment:
If I view the source, the indentation looks different? see: https://github.com/matabele/TiddlyWiki5/blob/351087c38be971047a701dfe2e9b0728a8b3247a/core/modules/filters/x-listops.js#L164 and https://github.com/matabele/TiddlyWiki5/blob/351087c38be971047a701dfe2e9b0728a8b3247a/core/modules/filters/x-listops.js#L211